PyInstaller/building/osx.py

```diff
@@ -189,14 +193,30 @@ def assemble(self):
         #
         # Move all directories from ./MacOS/ to ./Resources and create symlinks
         # in ./MacOS.
-        bin_dir =os.path.join(self.name, 'Contents', 'MacOS')
-        res_dir =os.path.join(self.name, 'Contents', 'Resources')
-        # Qt plugin directories does not contain data files.
-        ignore_dirs = set(['qt4_plugins', 'qt5_plugins'])
-        dirs = os.listdir(bin_dir)
-        for d in dirs:
-            abs_d = os.path.join(bin_dir, d)
-            res_d = os.path.join(res_dir, d)
-            if os.path.isdir(abs_d) and d not in ignore_dirs:
-                shutil.move(abs_d, res_d)
-                os.symlink(os.path.relpath(res_d, os.path.dirname(abs_d)), abs_d)
+        bin_dir = os.path.join(self.name, 'Contents', 'MacOS')
+        res_dir = os.path.join(self.name, 'Contents', 'Resources')
+        for inm, fnm in links:
```

Comment:
Could you add more comments to this loop what it's doing or split it into subfunctions? I'm getting lost whey trying to understand it.